src/cmdlib.sh

```diff
@@ -384,6 +384,15 @@ EOF
     else
         rm -vf "${local_overrides_lockfile}"
     fi
+
+    mkdir -p ${tmp_overridesdir}/contentsetrootfs/usr/share/buildinfo/
+    echo `jq . /srv/buildinfo/content_manifest.json` > ${tmp_overridesdir}/contentsetrootfs/usr/share/buildinfo/content_manifest.json
+    echo -n "Committing ${tmp_overridesdir}/contentsetrootfs... "
+    cat ${tmp_overridesdir}/contentsetrootfs/usr/share/buildinfo/content_manifest.json
+    ostree commit --repo=$tmprepo --tree=dir=${tmp_overridesdir}/contentsetrootfs -b contentset 
+    cat >> "${override_manifest}" << EOF
+  - contentset
+EOF
```

Comment:
I think this whole section of code should be conditional on the presence of `content_manifest.json`

Reply:
I'm not sure how this code is supposed to work?  The `content_manifest.json` is cat'ed, then an ostree commit is made, and the override manifest YAML doc has an empty `-contentset` entry added to it?

Reply:
Yes, pretty much but `cat content_manifest.json ` was not necessary. So basically just ostree commit the json file to /usr/share/ and adding `-contentset` entry to manifest yaml doc.